src/util/extent.js

```diff
@@ -128,9 +128,9 @@ export default function() {
 
     extents.include = function(x) {
         if (!arguments.length) {
-            return extraPoint;
+            return extraPoints;
         }
-        extraPoint = x;
+        extraPoints = x;
```

Comment:
@chrisprice mentioned this earlier https://github.com/ScottLogic/d3fc/issues/911#issuecomment-188778371

> If a value is stored internally as an array then I think it should be set as an array.